### DISCUSSION TITLE
RTX 2080ti + b650M

### DISCUSSION THREAD

**Discussion by bebelllm** [terminatorul/NvStrapsReBar#40] [General]
Hi, I have a motherboard b650m with already the resizebar support, I activated it, I activated above 4G support.
On windows 11 I launch NvStrapsReBar.exe
I set a targetsize to 16gb I saved. I reboot,
On NvStrapsReBar.exe the target size is still 16g and the current bar size still 256mb
On gpu-z it's still say PCI-Express Resizable BAR Resisable BAR Disabled
Gpu hardware support NO
Resisable BAR enabled in BIOS : Unsupported GPU

So I don't know what to do from here.... Thank you.

**Comment by Black6spdZ** [comment]
Doesnt work like that, you still have to mod your bios with nvstrapsrebar

**Comment by bebelllm** [comment]
Ok thank you, So I succeed, on asus motherboard you need to use flashback, because in the bios it doesn't work. I saw 15 fps / 10% performance increase in the game I play. Thank you.

**Reply by terminatorul** [reply]
This is a huge gain that most people never get to see !
Is this gain accurate ?

Can you disable ReBAR again, using NvStrapsReBar.exe, and show some screenshots with your FPS before and after ReBAR ?

**Reply by bebelllm** [reply]
Well I cannot reproduce the fps when I was posting this message, I was sure I went from 160 to 175. But When I tried to make it as scientific as possible I went from 166 to 171. So yes not quite as big ...
![Capture d’écran (24)-min](https://github.com/terminatorul/NvStrapsReBar/assets/11633307/1d7c498f-4ce2-4464-b94a-b5147908b8d7)
![Capture d’écran (26)-min](https://github.com/terminatorul/NvStrapsReBar/assets/11633307/3680f683-f5db-4996-a4c1-3d4136455746)

**Reply by terminatorul** [reply]
ty

**Reply by bebelllm** [reply]
@terminatorul ok so I didn't setup nvidia profile inspector properly before, I put like alsmost a random value in size, and now I put 8gb with 0x0000000200000000    while trying to find how to put 16gb and now I gain Inssaaanneeee performance. This mod transform my 2080ti in 3080ti .... WTF !!! Thank you.
![Capture d’écran (47)-min](https://github.com/terminatorul/NvStrapsReBar/assets/11633307/8e321910-d4c2-415d-8804-783d261054c4)

**Reply by terminatorul** [reply]
This must be the biggest improvement reported so far. Is this Call of Duty Warzone ?

**Reply by bebelllm** [reply]
yes it's warzone, but it's really strange, I have changed some setting in nvidia profile inspector, and I can't get back to 200 fps. And the GPU don't go back to 100% Like in previous picture. So yeah I really need to understand what are the perfect settings.
